### PR TITLE
Fix sending issue when socket disappears periodically

### PR DESCRIPTION
**Pull request by talien**

### CODE REVIEW

**Comment by talien** [comment]
Can you please review this request? We are using elasticbeanstalk and the logging container gets updated after the app container, which means that the logging socket disappears for a while.

**Comment by santigimeno** [comment]
@talien sorry, I had missed it. I'll try to review it in the following days. Thanks!

File: test/unix-connect-test.js [hunk 1]
```diff
@@ -88,4 +88,37 @@ vows.describe('unix-connect').addBatch({
       transport.close();
     }
   }
+}).addBatch({
+  'Logging works if server comes up again': {
+    topic: function () {
+      var self = this;
+      var n = 2;
```
Comment (santigimeno) [inline test/unix-connect-test.js]: why initialize to 2 instead of 0?

**Comment by santigimeno** [comment]
Can you rebase into a single commit?
Other than that LGTM with one comment/question. Thanks!

File: test/unix-connect-test.js (hunk 1 above)
Comment (talien) [inline test/unix-connect-test.js]: At that time, the message in buffer is "data3", and the next message will be data 4, so at that time, it seemed convenient to set n to 2, to be synced with the "times" global variable (which represents the number added to the message).

**Comment by santigimeno** [comment]
Landed in https://github.com/winstonjs/winston-syslog/commit/16185f5ce796fc6ba864316dbfdf4745e5c4212e. Published as `1.2.3`.

**Comment by santigimeno** [comment]
Waiting for @indexzero to publish it. Thanks!

**Pull request closed without merging**